packages/views/organizations.jade

```diff
@@ -1,21 +1,21 @@
 template(name="organizations")
   if currentUser
-    .container
-      .row
-        .col-sm-12
-          h2 Organizations
-          a.btn.btn-default.new-organization-link(href="{{path route='newOrganization'}}") Create
+    .container.view-header
+      h2.inline Organizations
```

Comment:
Definitely should be an h1...should have updated it when adding the class. My bad.